airflow/auth/managers/utils/fab.py

```diff
@@ -35,6 +35,7 @@
     "POST": ACTION_CAN_CREATE,
     "GET": ACTION_CAN_READ,
     "PUT": ACTION_CAN_EDIT,
+    "PATCH": ACTION_CAN_EDIT,
```

Comment:
```suggestion
```

Discussion is going on in https://github.com/apache/airflow/pull/47138 . Without this change, the PR can't be merged until that conversation on the ticket is resolved.

Reply:
This is what we will most likely do indeed!

Reply:
That's what was done in the legacy api